src/logic/embedding.lean

```diff
@@ -194,11 +194,11 @@ def punit {β : Sort*} (b : β) : punit ↪ β :=
 ⟨λ _, b, by { rintros ⟨⟩ ⟨⟩ _, refl, }⟩
 
 /-- Fixing an element `b : β` gives an embedding `α ↪ α × β`. -/
```

Comment:
I had no idea these existed, the names aren't at all predictable when starting from `prod.mk`.